app/components/code-search.js

```diff
@@ -116,6 +118,7 @@ export default Component.extend({
         return {
           addon: addons.findBy('name', result.addonName),
```

Comment:
I think you can get the score off of the addon here

Reply:
There's also a latestVersionDate on addon, if you want to do the other half of #99